examples/security-token/assets/controller.py

```diff
@@ -0,0 +1,222 @@
+from pyteal import *
+
+def approval_program():
+    """
+    This smart contract acts as a controller of the token. It
+    ensures that the permissions smart contract is called upon each
+    token transfer between non-reserve accounts. The permissions smart contract
+    validates the transfer with some rule checks (eg. asset_amount <= 100).
+    Controller application id is set in clawback-escrow (as a template parameter),
+    to ensure there is a call to controller during a token-transfer.
+    It also handles kill_status of the token. If token is killed then no token transfers
+    are allowed. User can only optOut of the asset.
+    """
+
+    var_is_killed = Bytes("is_token_killed")
+    var_total = Bytes("total_rules")
+    token_id = Bytes("token_id")
+    permissions_manager = Bytes("permissions_manager")	# permissions manager (asa.manager by default)
+    permission_id = Bytes("permissions_sc_app_id") # permissions smart contract application index
+
+    # Always verify that the RekeyTo property of any transaction is set to the ZeroAddress
+    # unless the contract is specifically involved ina rekeying operation.Btoi
+    no_rekey_addr = Txn.rekey_to() == Global.zero_address()
+
+    # retreive asset manager from Txn.ForeignAssets[0]
+    assetManager = AssetParam.manager(Int(0))
+
+    on_deployment = Seq([
+        assetManager, # load asset manager from store
+		Assert(And(
+			Txn.application_args.length() == Int(1),
+			# Txn.assets.length() == Int(1), [TEALv3]
+			no_rekey_addr,
+
```

Comment:
Please check the indentation here

Reply:
Yes, on vscode it looks fine on my system. But here it looks incorrect. Checking on another editor.